### ISSUE TITLE
Update hlf operator version on krew

### ISSUE THREAD

**Issue by ZappaBoy** [kfsoftware/hlf-operator#51]
The latest version of "hlf" operator on krew is still "v1.5.0". This old version not work following the "Getting started", more precisely, there is an error in "Preparing a connection string for the ordering service". In fact the last command uses the "admin-ordservice.yaml " that has a formatting error at line 12:

```shell
$ kubectl hlf utils adduser --userPath=admin-ordservice.yaml --config=ordservice.yaml \
        --username=admin --mspid=OrdererMSP
Error: yaml: line 12: found character that cannot start any token
```

 I think that upgrade to version "1.5.1" will solve this issues.

-----
Update: I can confirm that updating to version 1.5.1 solve the issue. Tested in #52.
There is still an error in "Getting Started" when trying to install the chaincode. I open a new issue about this.

**Comment by dviejokfs** [comment]
Hi @ZappaBoy 

Yes, 1.5.1 should resolve the issues.

The issue was solved in this #48 

You can try to update the hlf plugin and let us know.

**Comment by dviejokfs** [comment]
Hi @ZappaBoy 

I just tested it using HLF v1.5.1 and it's working for me.

You'll need to generate the `ordservice.yaml` file since the bug was with the template of the network config.

Please reopen if the issue still exists. 

You'll need to upgrade the hlf plugin:
```bash
kubectl krew upgrade hlf
```
